### PR TITLE
[scaling] Scaling of statusbar

### PR DESCRIPTION
**Pull request by eekkelund**
Works on Nexus5

### CODE REVIEW

File: src/qml/Statusbar.qml [hunk 1]
```diff
@@ -38,7 +38,7 @@ import MeeGo.Connman 0.2
 Item {
     id: root
     z: 201
-    height: 48
+    height: root.width/12
```
Comment (faenil) [inline src/qml/Statusbar.qml]: remove "root" here, I think it's a bit confusing, there is no need if it's a property of the same object

File: src/qml/Statusbar.qml [hunk 2]
```diff
@@ -138,9 +138,9 @@ Item {
         StatusbarItem {
             Label {
                 id: tech
-                width: 16
-                height: 16
-                font.pointSize: 6
+                width: root.height/4
+                height: root.height/4
+                font.pointSize: root.height/14
```
Comment (faenil) [inline src/qml/Statusbar.qml]: why pointSize? does this guarantee that it will fit in the Label box?

File: src/qml/Statusbar.qml [hunk 3]
```diff
@@ -192,23 +192,23 @@ Item {
             source: "image://theme/icon_gps_normal"
         }
         StatusbarItem {
-            source: "image://theme/icon_play_pause"
+            source: "image://theme/icon_play_pause"
```
Comment (faenil) [inline src/qml/Statusbar.qml]: remove the additional whitespace

File: src/qml/Statusbar.qml [hunk 4]
```diff
@@ -192,23 +192,23 @@ Item {
...
                 id: minutes
                 anchors.top: hours.bottom
-                anchors.topMargin: 4
-                width: 16
-                height: 16
-                font.pointSize: 6
+                anchors.topMargin: root.height/8
+                width: root.height/4
+                height: root.height/4
+                font.pointSize: root.height/14
```
Comment (faenil) [inline src/qml/Statusbar.qml]: same as above

File: src/qml/StatusbarItem.qml [hunk 5]
```diff
@@ -9,6 +9,8 @@ Item {
     Layout.fillHeight: true
     Image {
         id: icon
+        width: root.height/2
```
Comment (faenil) [inline src/qml/StatusbarItem.qml]: please do not refer to items that are not defined in this QML file :)

You can either add an "iconSize" property that you set when creating a StatusbarItem, or you can have a property that you then initialize with the component that creates StatusbarItem

Let me know if you need help :)

**Comment by faenil** [comment]
left some comments :)

File: src/qml/Statusbar.qml (hunk 1 above)
Comment (eekkelund) [inline src/qml/Statusbar.qml]: Ok thanks.:) Yeah that is the problem I don't myself have low resolution device to test with.

**Comment by eekkelund** [comment]
I haven't  tested with low resolution device..

**Comment by locusf** [comment]
LGTM merging

**Pull request merged**